core/benches/blocks/common.rs

```diff
@@ -136,10 +172,12 @@ fn restore_every_nth(
     Ok(instructions)
 }
 
-fn build_wsv(account_id: &AccountId, key_pair: &KeyPair) -> WorldStateView {
+pub fn build_wsv(account_id: &AccountId, key_pair: &KeyPair) -> WorldStateView {
     let kura = iroha_core::kura::Kura::blank_kura_for_testing();
     let mut wsv = WorldStateView::new(World::with([], BTreeSet::new()), kura);
     wsv.config.transaction_limits = TransactionLimits::new(u64::MAX, u64::MAX);
+    wsv.config.wasm_runtime_config.fuel_limit = u64::MAX;
+    wsv.config.wasm_runtime_config.max_memory = u32::MAX;
```

Comment:
Is this necessary even after fixing the memory leak? Or is there just no sense to limit these in a benchmark situation? :thinking:

Reply:
> Or is there just no sense to limit these in a benchmark situation? 🤔

Exactly, in the benchmark we are submitting large block that won't normally pass limitations in the validator.